### DISCUSSION TITLE
Any support with stripe's new element: payment element

### DISCUSSION THREAD

**Discussion by HaodongZhai** [swellstores/community#299] [Ideas]
we are plan to use strile's payment element, but only found supoort with stripe's card element. 
Here found a discussion: https://github.com/orgs/swellstores/discussions/183, but no more reply.
Are there have any other conculsion with stripe's payment element?

**Comment by sags95** [comment]
@HaodongZhai We don't have any current plans to support the Payment Element through swell-js at this time, but there is an integration pathway nonetheless. The key thing is that you would have to handle the creation and confirmation of paymentIntents outside of the `tokenize()` method when using the element.

The other important thing to mention is that not all payment methods support a `manual` capture method, so you might be restricted to only using a subset of them (if you still want to capture the payments through Swell).

The steps would be the following:

1. Follow Stripe's implementation guide [here](https://stripe.com/docs/payments/quickstart)
2. Create paymentIntents with `capture_method: 'manual'` and `setup_future_usage: 'off_session'`, this is required for Swell to capture the paymentIntent in the `submitOrder()` method.
3. In the confirmation step, attach the paymentIntent to the cart
```
        await swell.cart.update({
            billing: {
               card: {
                   token: <pm_***>
                },
                intent: {
                    stripe: {
                        id: <pi_***>
                    }
                }
            }
        })
```
4. Submit cart
